### ISSUE TITLE
Client randomly hangs on "Placing..."

### ISSUE THREAD

**Issue by EricRabil** [dynasticdevelop/place#180]
This is a bug reported by Aaron from the Place Discord:
```
Sometimes, it'll say it's placing a pixel for waaay too long and I'll have to refresh the page and re-place the pixel
```
This is a developing issue and more information will be added soon.

**Comment by Hexzatite** [comment]
This happens sometimes too. It'll just "hang" on the 'Placing...' text for like, 10 seconds or until i reload. i can't get it to happen reliably though, seems to be random.

**Comment by AppleBetas** [comment]
Either the server is hanging on providing a response, or its just a lapse in connectivity and the client isn't timing out properly. This will need some investigation

**Comment by AppleBetas** [comment]
Also, I should add that this may be related to #181 because the server may be crashing, causing the request to not work.

**Comment by AppleBetas** [comment]
Seems to be fixed by the same cause of #181
